src/Dropdown.js

```diff
@@ -182,15 +120,7 @@ class Dropdown extends React.Component {
         dropup: dropup
       }
     ), cssModule);
-
-    return (
-      <Tag
-        {...attributes}
-        className={classes}
-      >
-        {this.renderChildren()}
-      </Tag>
-    );
+    return <Manager {...attrs} className={classes}>{}</Manager>;
   }
 }
 
```

Comment:
Codewise, even cleaner and easier to read. Removed unnecessary logic in a few areas. 👍